### DISCUSSION TITLE
2023-21: 我的 1Password 密钥管理实践

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#405] [Announcements]
# 2023-21: 我的 1Password 密钥管理实践

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2023-21/

**Comment by Harry-Chen** [comment]
然而自从某个版本（大概一两个月前？）开始 1P 在 macOS 上的 SSH Agent 就不工作了，表现为授权了也显示 refused，于是只好先扔下等它修。

**Reply by Xuanwo** [reply]
难绷😂

**Comment by STRRL** [comment]
> 申请 1Password 开源赞助

这个好棒! 我们最近也遇到了 Chaos Mesh 的 CI Infrastructure 集群的 credentials 只有俺和 Keao 有 access, 也有单点问题.

后面俺们也去申请下!
